daemon/delete.go

```diff
@@ -25,7 +26,7 @@ func (daemon *Daemon) ContainerRm(name string, config *ContainerRmConfig) error
 			// TODO: why was just job.Error(err) without return if the function cannot continue w/o container name?
 			//job.Error(err)
 		}
-		parent, n := path.Split(name)
+		parent, n := filepath.Split(name)
```

Comment:
I think you're right. I'm going through file by file with the changes to the daemon which I have running on Windows today - I think earlier on I was a bit overzealous on the fixing of path/filepath. I'll amend that.